### ISSUE TITLE
Language ISO Code of External Subtitles are removed after rename

### ISSUE THREAD

**Issue by wsegatto** [Sonarr/Sonarr#4336]
**Describe the bug:**
When you rename series via the "Rename" feature and confirm, the subtitles are supposed to be renamed as well.
However, my pt-BR subtitles are not getting renamed correctly. The language "Language ISO Code" is lost. 
By the way, this was also reported for Radarr, [link](https://github.com/Radarr/Radarr/issues/5191).

Before rename:
```
 Sex.and.the.City.S01E01.Sex.and.the.City.x264.DVD.en.srt
 Sex.and.the.City.S01E01.Sex.and.the.City.x264.DVD.mkv
 Sex.and.the.City.S01E01.Sex.and.the.City.x264.DVD.pt-BR.srt <<<<<
```

After rename:
```
 Sex.and.the.City.S01E01.Sex.and.the.City.x264.AAC.DVD.en.srt
 Sex.and.the.City.S01E01.Sex.and.the.City.x264.AAC.DVD.mkv
 Sex.and.the.City.S01E01.Sex.and.the.City.x264.AAC.DVD.srt <<<<<
```

This wouldn't be a serious issue, however Bazarr detects these subtitles with no Language ISO Code as "pt", not "pt-BR", which causes it to download the pt-BR subtitles again.

![image](https://user-images.githubusercontent.com/35738729/108366594-50a79780-71d7-11eb-9a05-f229d3981481.png)

**To Reproduce**
1. Have an episode with an EN and a pt-BR SRT subtitle.
2. Make sure this episode is not correctly named based on your media naming rules.
3. Go to this series and click "Rename"
4. Confirm the change, renaming the file
5. Check the new subtitle name, the ".pt-BR" part of the filename is missing.

**Expected behavior**
Subtitles should keep their language codes (in my case, the "pt-BR" is lost).

**Platform Information:**
OS: Raspbian 10 arm64
Version: 3.0.4.1130
Package Version: 3.0.4 by Team Sonarr
Mono Version: 5.18.0.240

**Comment by markus101** [comment]
This will be covered by https://github.com/Sonarr/Sonarr/issues/2570
